stix_shifter_modules/azure_sentinel/stix_translation/json/alert_from_stix_map.json

```diff
@@ -169,7 +109,62 @@
       "dst_geolocation": ["networkConnections.destinationLocation"],
       "src_application_user_ref.user_id":["userStates.aadUserId"],
       "src_application_user_ref.type":["userStates.logonType"],
-      "time_observed": ["lastModifiedDateTime"]
+      "time_observed": ["lastModifiedDateTime"],
+      "x_activityGroupName": ["activityGroupName"],
+      "x_assignedTo": ["assignedTo"],
+      "x_comments": ["comments"],
```

Comment:
I wonder if this should go in the `description` property. Either this or the feedback field.

Reply:
there's already description one to one mapping:`"description": ["description"]`. i would like to leave comments as custom object.